modules/xmpp/JingleSessionPC.js

```diff
@@ -57,6 +58,92 @@ function getEndpointId(jidOrEndpointId) {
     return Strophe.getResourceFromJid(jidOrEndpointId) || jidOrEndpointId;
 }
 
+/**
+ * Add "source" element as a child of "description" element.
+ */
+function _addSourceElement(description, s, ssrc_) {
+    const val = `${s.owner} ${s.owner}`; // $ what should we set this to?
```

Comment:
I don't think it's appropriate to use the owner of the source here since the owner gets remapped but these msid values will be used in the remote SDP until the peerconnection gets closed. Can we generate these values here ? something like `remote-video-1`, `remote-video-2` and so on ?

Reply:
Now using serial number (per media type) to create msid attribute.